kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionManager.java

```diff
@@ -208,6 +210,15 @@ public void start() {
         Logger log = session.getLogger();
         log.status("Using Kubernetes at: " + client.getMasterUrl());
         createNamespace();
+
+        // generate fabric8 resources using build and resource goals
```

Comment:
Maybe instead of a comment here explaining what is used and how we could rework `ResourceGeneratorBuilder` to be more specific of what it really does? As it's anyway very much related to `fabric8` plugin we could equally well rename the class too.